astro.config.mjs

```diff
@@ -23,7 +23,7 @@ export default defineConfig({
   image: {
     service: passthroughImageService(),
   },
-  site: 'https://experienceleague.adobe.com',
+  site: 'https://experienceleague.adobe.com/developer/commerce/storefront/',
```

Comment:
Have you thoroughly tested whether this works with our `basePath` implementation?